src/ippair.c

```diff
@@ -773,6 +773,22 @@ static IPPair *IPPairGetUsedIPPair(void)
     return NULL;
 }
 
+int GetFlowAddresses(const Flow *f, Address *ip_src, Address *ip_dst)
```

Comment:
Lost inlining because it is now used by 2 files...

Reply:
Not critical I think

Reply:
ok